### ISSUE TITLE
Twitch iframes are filtered out by sanitizer

### ISSUE THREAD

**Issue by humphd** [Seneca-CDOT/telescope#2897]
<img width="1033" alt="Screen Shot 2022-02-12 at 9 29 50 AM" src="https://user-images.githubusercontent.com/427398/153715345-1ef5b9f3-4569-40cc-82b2-996f1bf46315.png">

We need to update the sanitizer to allow the twitch iframe origin.

**Comment by humphd** [comment]
That's for the front-end, we also need to fix https://github.com/Seneca-CDOT/telescope/blob/master/src/backend/utils/html/sanitize.js#L50-L57 in the backend.  And the parser service would need this too.

**Comment by HyperTHD** [comment]
We also need to update the tests as well to ensure that this is working as intended. This is where they should go https://github.com/Seneca-CDOT/telescope/blob/master/test/sanitize-html.test.js

**Comment by HyperTHD** [comment]
Taking this

**Comment by humphd** [comment]
I'm seeing this for a second time, still unfixed.  @HyperTHD do you have time to work on this, or should we give to someone else?

**Comment by HyperTHD** [comment]
> I'm seeing this for a second time, still unfixed. @HyperTHD do you have time to work on this, or should we give to someone else?

Been pretty busy lately, but got some time. I'll have a PR up today hopefully for this

**Comment by manekenpix** [comment]
The issue is still there
![image](https://user-images.githubusercontent.com/23108901/155035782-ae8ae4db-92e6-4241-90ff-b95149ef5db2.png)

**Comment by humphd** [comment]
Won't it need to re-index for this to get applied to that post?

**Comment by manekenpix** [comment]
@humphd or, I forgot, let me make the changes on staging to see if it solves the issue

**Comment by manekenpix** [comment]
No, still the same issue
![image](https://user-images.githubusercontent.com/23108901/155037855-29f4703e-f56f-4696-8400-42cafc2ea1ed.png)

**Comment by manekenpix** [comment]
I think this is probably related to @2306, I see the same behaviour in both cases.

**Comment by DukeManh** [comment]
Could be because the hostname is supposed to be `player.twitch.tv` vs `twtich.tv`
@HyperTHD

**Comment by DukeManh** [comment]
This is Alex's twitch iframe


```html
<iframe src="https://player.twitch.tv/?video=1303803789&amp;parent=dev.to&amp;autoplay=false" allowfullscreen="true" scrolling="no" width="710" height="399" frameborder="0">
</iframe>
```

**Comment by HyperTHD** [comment]
> This is Alex's twitch iframe
> 
> ```
> <iframe src="https://player.twitch.tv/?video=1303803789&amp;parent=dev.to&amp;autoplay=false" allowfullscreen="true" scrolling="no" width="710" height="399" frameborder="0">
> </iframe>
> ```

I tested that hostname, still not viewing it locally

**Comment by humphd** [comment]
<img width="923" alt="Screen Shot 2022-02-21 at 7 58 05 PM" src="https://user-images.githubusercontent.com/427398/155044259-844fa3eb-40f7-4b74-b4bd-723ac0596ce7.png">

For me on staging, I don't get a URL on the iframe, what am I missing?  Did we try re-indexing?

**Comment by manekenpix** [comment]
@humphd yes, all the feeds were re-indexed on staging, but the issue persists.

**Comment by humphd** [comment]
@HyperTHD  I'd say that it's the sanitizer using `'www.twitch.tv'` vs. `'player.twitch.tv'` and stripping it during parsing.

**Comment by HyperTHD** [comment]
I'm not able to see any changes when switching it out, but I can put up a PR to fix this, and hopefully re-indexing on staging will fix that
